WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/quicklinkribbons/QuickLinkRibbonBuilder.kt

```diff
@@ -1,6 +1,7 @@
 package org.wordpress.android.ui.mysite.cards.quicklinkribbons
```

Comment:
Nitpick: Not related to the changes but I noticed that pluralized name `quicklinkribbons` is used in the package name. It can be renamed to `quicklinksribbon`. Notice pluralized "links" and singular "ribbon".

Few other places in the codebase where pluralized `ribbons` still being used that we might want to rename:

`org.wordpress.android.ui.mysite.cards.CardsBuilder `-> `quickLinkRibbonsBuilderParams`

`org.wordpress.android.ui.mysite.cards.CardsBuilderTest` -> 
`quickLinkRibbons`
`quickLinkRibbonsBuilderParams`
`initQuickLinkRibbons`
`given tabs enabled, when cards are built, then quick link ribbons built`

Reply:
@ashiagr Thanks for looking into the package name and the occurrences of pluralized `ribbons`. Your observational skills are as always superb. 🙌🏼  I have done the necessary changes in 2f7963e. Please let me know if I need to make any additional changes. I believe we could update all the occurrences of`quickLinkRibbon` to `quickLinksRibbon`. But I think we can take it up as a separate PR.  Wdyt?. 🤔

Reply:
Sounds good! Feel free to merge this PR when you're ready.